src/utilities.h

```diff
@@ -662,4 +675,18 @@ ConnectionMap createConnectionMap(const VariablePtr &variable1, const VariablePt
  */
 std::vector<VariablePtr> equivalentVariables(const VariablePtr &variable);
 
+/**
+ * @brief Test the given entities are equal to entities in owner.
+ *
+ * Test to see if all the entites given in @p entities are equal to
+ * entities in @p owner.  The order that the entities appear in is not
```

Comment:
entities cf entites
@p owner in brief, @p entities in brief

Reply:
6a63514